torchvision/prototype/transforms/_geometry.py

```diff
@@ -460,31 +476,29 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
 
         top = torch.randint(0, height - output_height + 1, size=(1,)).item()
         left = torch.randint(0, width - output_width + 1, size=(1,)).item()
-        return dict(top=top, left=left, height=output_height, width=output_width)
 
-    def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
-        return F.crop(inpt, **params)
+        return dict(
+            top=top, left=left, height=output_height, width=output_width,
+            input_width=width, input_height=height
+        )
```

Comment:
In other words we see that `_get_params` precomputed image size for further usage with images, bboxes etc